hw9/Task_1.ts

```diff
@@ -0,0 +1,23 @@
+// Task 1
+// Создайте класс Animal, в конструктор которого передается имя,
+// и с помощью метода данного класса можно вывести фразу `This is a <animalType>. It's name is <name>`.
+// Так же данный класс должен иметь статическое поле animalType,
+// значение которого должно подставляться в фразу, которую выводит метод выше.
+
+class Animal {
+    static type: string = 'frog';
+    name: string;
+
+    constructor(type: string, name: string) {
```

Comment:
we shouldn't use type as constructor parametr

Reply:
Fixed.